plugins/shared/proto/v1/shared.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+package hashicorp.nomad_autoscaler.plugins.shared.proto.v1;
+option go_package = "proto";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+message ScalingAction {
+    int64 count = 1;
+    string reason = 2;
+    bool error = 3;
+    ScalingDirection direction = 4;
+    google.protobuf.Any meta = 5;
+}
+
+enum ScalingDirection {
+    SCALING_DIRECTION_UNSPECIFIED = 0;
```

Comment:
what's `unspecified` for? looks like most of its uses involve an error?

Reply:
It is [linter rule](https://docs.buf.build/lint-rules#enum_zero_value_suffix), so it is there only to fulfil that.